_docs/results/duckassist.md

```diff
@@ -0,0 +1,15 @@
+---
+title: What is DuckAssist on DuckDuckGo Search?
+nav_title: DuckAssist
+description: DuckAssist is an optional feature on DuckDuckGo Search that anonymously generates answers to queries based on information from the web.
+category: Search Results
+order: 1125
+---
+
+DuckAssist is an optional feature in our search results that can anonymously generate answers to search queries. To do this, it scans the web for relevant content and then uses AI-powered natural language technology to generate a brief answer based on relevant information found. DuckAssist responses always link directly to one or two sources, citing where the answer came from, so you can easily go and get more detailed information. It's important to remember that responses may contain inaccuracies because they're auto-generated from cited sources and based on <a href="{{ site.baseurl }}/results/duckassistbot/">crawling the web</a>.
+
+DuckAssist cannot answer follow-up questions, however, we also offer <a href="{{ site.baseurl }}/aichat">DuckDuckGo AI Chat</a>, which is a private AI-powered chat service that supports chat models from OpenAI, Anthropic, and more.
```

Comment:
Just wanted to confirm that you intended to link to the AI Chat help page here. If you meant to link to AI Chat itself, the `href` should be updated to be `https://duckduckgo.com/aichat` instead of `{{ site.baseurl }}/aichat`.